kubernetes-deploy.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+# Set up Corda on Kubernetes; must be run from the root of corda-runtime-os
+
+set -e
+
+./gradlew publishOSGiImage
+
+# Currently the init containers are not repeatble, so ensure we start from scratch each time
+kubectl delete namespace corda || echo corda namesapce already deleted?
+kubectl create namespace corda
+
+# Usual prereqs
+helm install prereqs -n corda oci://registry-1.docker.io/corda/corda-dev-prereqs --timeout 10m --wait
+
+# Corda
+helm dependency build charts/corda
+helm upgrade --install corda -n corda charts/corda \
```

Comment:
nit: shouldn't matter given we're cleaning out the namespace anyway but just to make a point...
```suggestion
helm install corda -n corda charts/corda \
```

Reply:
done